client/src/components/publications/importedPublication/ImportedPublication.js

```diff
@@ -30,27 +35,33 @@ const ImportedPublication = ({pub, index, setChecked}) => {
         setExpand(expand)
```

Comment:
Button looks kinda ugly haha 
![image](https://user-images.githubusercontent.com/40584352/124740718-63d8bb00-df5e-11eb-9cc0-5794f156dc6c.png)
But good point to bring up, I kinda want to show that it's a clickable link like highlight it in blue or something. I'll note it down for next semester

Reply:
Actually, I guess we can keep it since it's a third-party website (not a page of Researchify).

Here are some potential solutions I can think of, which might be possible to apply to your case:  
1. Use `href` tag in `<a>`,   and the link will act like a normal link. [Example](https://www.w3schools.com/tags/tag_a.asp)
2. Use Button from React-Bootstrap, with `variant="link"`, to make to button looks like a link. [Example](https://react-bootstrap.github.io/components/buttons/#:~:text=use%20any%20of%20the%20available%20button%20style%20types%20to%20quickly%20create%20a%20styled%20button.%20just%20modify%20the%20variant%20prop.)

But I think we can leave it for now as it wasn't that important  :)

Reply:
I'll go with href tag cause it's nice and simple:
![image](https://user-images.githubusercontent.com/40584352/124862504-d8603800-dff8-11eb-8046-d87aed39e646.png)

Thanks!